org.mozilla.Firefox.json

```diff
@@ -67,6 +67,17 @@
                 "install -Dm 644 -t /app/firefox/data endless-default-prefs.js",
                 "install -Dm 644 -t /app/firefox/data distribution.ini"
             ],
+            "x-checker-data": {
+                "type": "firefox",
+                "browser": {
```

Comment:
As described on https://github.com/endlessm/flatpak-external-data-checker/pull/28 I think it should be enough to specify `"type": "firefox"` – this checker is completely Firefox-specific so I don't see much value in generalizing the filenames.